Composer/packages/ui-plugins/select-dialog/src/SelectDialog.tsx

```diff
@@ -1,15 +1,19 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import React, { useState } from 'react';
+import React, { useCallback, useState } from 'react';
 import { IComboBoxOption, SelectableOptionMenuItemType } from 'office-ui-fabric-react/lib/ComboBox';
 import { FieldProps, useShellApi } from '@bfc/extension-client';
 import formatMessage from 'format-message';
+import { IntellisenseTextField } from '@bfc/adaptive-form';
+import { WithTypeIcons } from '@bfc/adaptive-form/lib/components/WithTypeIcons';
```

Comment:
This looks like it should be exported from the main module of `@bfc/adaptive-form`.